iothub/service/src/MessageSystemPropertyNames.cs

```diff
@@ -8,40 +8,104 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public static class MessageSystemPropertyNames
     {
+        /// <summary>
+        /// A user-settable identifier for the messages. If this value is not supplied by the user,
+        /// the service client will set this to a new GUID.
+        /// </summary>
         public const string MessageId = "message-id";
 
+        /// <summary>
+        /// LockToken of the received message.
+        /// </summary>
         public const string LockToken = "iothub-messagelocktoken";
 
+        /// <summary>
+        /// A number (unique per device-queue) assigned by IoT Hub to each cloud-to-device message.
+        /// </summary>
         public const string SequenceNumber = "iothub-sequencenumber";
 
+        /// <summary>
+        /// A destination specified in Cloud-to-Device messages.
+        /// </summary>
         public const string To = "to";
 
+        /// <summary>
+        /// Number of times the message has been previously delivered.
+        /// </summary>
         public const string DeliveryCount = "iothub-deliverycount";
 
+        /// <summary>
+        /// Date and time when the message was received by the server.
+        /// </summary>
         public const string EnqueuedTime = "iothub-enqueuedtime";
 
+        /// <summary>
+        /// Date and time of message expiration.
+        /// </summary>
         public const string ExpiryTimeUtc = "absolute-expiry-time";
 
+        /// <summary>
+        /// A string property in a response message that typically contains the MessageId of the request, in request-reply patterns.
+        /// </summary>
         public const string CorrelationId = "correlation-id";
 
+        /// <summary>
+        /// An ID used to specify the origin of messages. When messages are generated by IoT Hub, it is set to 'IoT Hub name'.
```

Comment:
Is it literally set to the string 'IoT Hub name' or the value of their IoT hub name?

Reply:
The value of the iot hub name.

Reply:
Is this going to be more clear instead (curly brackets)? {IoT Hub name}

Reply:
The fact it is in quotation marks implies, to me, that it is the value. I'd suggest rewriting.

Reply:
Yes, I'd go with simple English.